hs_dictionary/tests/test_universities.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class TestUniversities(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_universities_no_query(self):
+        response = self.client.get('/hsapi/dictionary/universities/', format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        content = json.loads(response.content)
+        self.assertEqual(len(content), 9363)
+
+    def test_universities_query(self):
+        response = self.client.get('/hsapi/dictionary/universities/dubai/', format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        content = json.loads(response.content)
+        self.assertEqual(len(content), 9)
```

Comment:
@aphelionz Instead of asserting the len(content) be 9, perhaps it is more flexible to assert a keyword be in the content string? The consideration for this is that if the universities csv file get changed for the test case, perhaps one or more characters are added or deleted, this test would fail even it returns the correct content.